updates/P6.txt

```diff
@@ -0,0 +1,135 @@
+                           Protocol update
+
+Protocol Version: 6
+Builds on Protocol Version: 5
+
+                               Abstract
+
+This document describes the changes in Concordium protocol version 6
+compared to protocol version 5.
+
+The protocol change has the following main effects
+- it changes the consensus protocol from an existing two-layer design
+  to a new Concordium BFT consensus
+- fixes the behaviour of state rollbacks in specific scenarios with
```

Comment:
I think Concordium uses American English, i.e., behavior. This also applies in other places below.

Reply:
True.